### PR TITLE
Unify class list helpers

### PR DESCRIPTION
**Pull request by jwunderl**
element classList helpers cleanup from https://github.com/microsoft/pxt/pull/5549

### CODE REVIEW

File: pxtlib/browserutils.ts [hunk 1]
```diff
@@ -958,4 +958,48 @@ namespace pxt.BrowserUtils {
...
+            return el.classList.contains(cls);
+        } else {
+            const classes = el.className.split(" ") as string[];
+            return !(classes.indexOf(cls) < 0)
+        }
+    }
+
+    export function addClass(el: SVGElement | HTMLElement, classes: string) {
+        classes
+            .split(" ")
```
Comment (pelikhan) [inline pxtlib/browserutils.ts]: \s+

File: pxtlib/browserutils.ts [hunk 2]
```diff
@@ -958,4 +958,48 @@ namespace pxt.BrowserUtils {
...
+                if (classes.indexOf(cls) < 0) {
+                    el.className.baseVal += " " + cls;
+                }
+            }
+        }
+    }
+
+    export function removeClass(el: SVGElement | HTMLElement, classes: string) {
+        classes
+            .split(" ")
```
Comment (pelikhan) [inline pxtlib/browserutils.ts]: \s+

File: pxtsim/runtime.ts [hunk 3]
```diff
@@ -4,28 +4,48 @@ namespace pxsim {
...
 
-        export function removeClass(element: HTMLElement, classes: string) {
-            if (!element) return;
-            if (!classes || classes.length == 0) return;
-            function removeSingleClass(el: HTMLElement, singleCls: string) {
-                if (el.classList) el.classList.remove(singleCls);
-                else el.className = el.className.replace(singleCls, '').replace(/\s{2,}/, ' ');
+        export function addClass(el: SVGElement | HTMLElement, classes: string) {
+            classes
+                .split(" ")
```
Comment (pelikhan) [inline pxtsim/runtime.ts]: \s+

File: pxtlib/browserutils.ts [hunk 4]
```diff
@@ -964,21 +964,21 @@ namespace pxt.BrowserUtils {
         if (el.classList) {
             return el.classList.contains(cls);
         } else {
-            const classes = el.className.split(" ") as string[];
+            const classes = el.className.split("\s+") as string[];
```
Comment (pelikhan) [inline pxtlib/browserutils.ts]: /\s+/

File: pxtlib/browserutils.ts [hunk 5]
```diff
@@ -964,21 +964,21 @@ namespace pxt.BrowserUtils {
...
             return !(classes.indexOf(cls) < 0)
         }
     }
 
     export function addClass(el: SVGElement | HTMLElement, classes: string) {
         classes
-            .split(" ")
+            .split("\s+")
```
Comment (pelikhan) [inline pxtlib/browserutils.ts]: /\s+/

**Comment by jwunderl** [comment]
Looks like my sim might have been unlinked by npm at some point, not sure how I missed that. Thanks

**Pull request merged**